org.gustavoperedo.FontDownloader.json

```
@@ -0,0 +1,59 @@
+{
+    "app-id" : "org.gustavoperedo.FontDownloader",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.36",
```

Update runtime to 3.38
```suggestion
    "runtime-version" : "3.38",
```

---

For some reason changing that line results in an error: 
```
bwrap: execvp fontdownloader: No such file or directory
```

Not really sure why, since going back to 3.36 get rid of the error...

---

Restarting GNOME Builder seems to have fixed the issue